src/parser/shaman/elemental/modules/talents/Stormkeeper.js

```diff
@@ -40,6 +47,22 @@ class Stormkeeper extends Analyzer {
     return this.damageDoneByBuffedCasts / (this.owner.fightDuration / 1000);
   }
 
+  on_byPlayer_applybuff(event){
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.STORMKEEPER_TALENT.id) {
+      this.stormkeeperCasts += 1;
+    }
+  }
+
+  on_fightend(event){
```

Comment:
Hawk recommended on my PR a different, newer style for event subscription. This is quite nice as it can handle some of the filtering for you.

```javascript
import Events from 'parser/core/Events';
import SPELLS from 'common/SPELLS';
import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';

constructor(...args) {
  this.addEventListener(Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.STORMKEEPER_TALENT), (event) => { ... });
  this.addEventListener(Events.fightend, (event) => { ... });
}
```